docs/machine-learning/tutorials/image-classification-api-transfer-learning.md

```diff
@@ -0,0 +1,456 @@
+
+#### Training phase
+
+Once the values from the frozen layers are computed, they are used as input to retrain the bottleneck layer. This process is iterative and runs for the number of times specified by model parameters. During each run, the loss and accuracy are evaluated and the appropriate adjustments are made to improve the model with the goal of minimizing the loss and maximizing the accuracy. Once training is finished, two models are output. One of them is the `.pb` version of the model and the other is the `.zip` ML.NET serialized version of the model. When working in environemnts supported by ML.NET, it is recommended to use the `.zip` version of the model. However, in environemnts where ML.NET is not supported, you have the option of using the `.pb` version.
+
+## Understand the pretrained model
+
+The pre-trained model used in this tutorial is the 101-layer variant of the Residual Network (ResNet) v2 model. The original model takes an input image of size 224 x 224 and outputs the class probabilities for each of the classes it's trained on.
+
+## Create console application
+
+Now that you have a general understanding of transfer learning and the Image Classification API, it's time to build the application
+
+1. Create a **.NET Core Console Application** called "DeepLearning_ImageClassification_API"
+1. Install the **Microsoft.ML 1.4.0-preview** NuGet Package:
+    1. In Solution Explorer, right-click on your project and select **Manage NuGet Packages**.
+    1. Choose "nuget.org" as the Package source.
+    1. Select the **Browse** tab.
+    1. Check the **Include prerelease** checkbox.
+    1. search for **Microsoft.ML**.
+    1. Select the **Install** button.
+    1. Select the **OK** button on the **Preview Changes** dialog and then select the **I Accept** button on the **License Acceptance** dialog if you agree with the license terms for the packages listed.
+    1. Repeat these steps for **Microsoft.ML.Dnn 0.16.0-preview** and **Microsoft.ML.ImageAnalytics 1.4.0-preview**.
+
+### Prepare and understand the data
+
+> [!NOTE]
+> The datasets for this tutorial are from Maguire, Marc; Dorafshan, Sattar; and Thomas, Robert J., "SDNET2018: A concrete crack image dataset for machine learning applications" (2018). Browse all Datasets. Paper 48. https://digitalcommons.usu.edu/all_datasets/48
+
+SDNET2018 is an image dataset that contains annotations for cracked and non-cracked concrete structures (bridge decks, walls, and pavement). 
+
+![](./media/image-classification-api-transfer-learning/crackeddeck.jpg)
```

Comment:
It would be good to have a cracked and non cracked image

Reply:
#resolved